repair_order_group/readme/USAGE.md

```
@@ -0,0 +1,3 @@
+Create the first repair order.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)